content/about/consortium.md

```diff
@@ -9,7 +9,7 @@ weight: 80
 - [INSTITUTE OF APPLIED BIOSCIENCES (*INAB*)](https://www.inab.certh.gr/)
 - [ETHNIKO KENTRO EREVNAS KAI TECHNOLOGIKIS ANAPTYXIS (*CERTH*)](https://www.certh.gr/root.en.aspx)
 - [BARCELONA SUPERCOMPUTING CENTER CENTRO NACIONAL DE SUPERCOMPUTACION (*BSC CNS*)](https://www.bsc.es/)
-- [ORGANISATION EUROPEENNE POUR LA RECHERCHE NUCLEAIRE (*CERN*)](https://home.cern/)
+- [EUROPEAN ORGANISATION FOR NUCLEAR RESEARCH (*CERN*)](https://home.cern/)
```

Comment:
I made this change mainly to align with the official names used in the proposal. However, I'm not sure if this is necessary, and honestly, I'm not fond of uppercase letters. @fpsom any comment please?

Reply:
Given that this is the EVERSE website, I'd leave the choice on how to actually write the name of each Organization up to them (we are also not putting in the official name in the proposal). So, I'd keep Graeme's suggestion in.

Reply:
Perfect, thank you! Maybe it would be useful to remind everyone to provide feedback on both the consortium and the preferred name for the website.